releasenotes/notes/make-statevector-subscriptable-and-add-inner-product_method-a0337393d9a5b666.yaml

```
@@ -0,0 +1,15 @@
+---
+features:
+  - |
+    Make :class:`qiskit.quantum_info.states.statevector.Statevector` subscriptable.
```

```suggestion
    Make :class:`~qiskit.quantum_info.states.statevector.Statevector` subscriptable.
```

`~` shortens things so they only display the last word.